Source/WebCore/accessibility/cocoa/AXTextMarkerCocoa.mm

```diff
@@ -54,7 +56,7 @@
         return;
     }
 
-    memcpy(&m_data, AXTextMarkerGetBytePtr(platformData), sizeof(m_data));
+    memcpySpan(asMutableByteSpan(m_data), unsafeMakeSpan(AXTextMarkerGetBytePtr(platformData), sizeof(m_data)));
```

Comment:
Hmmm this seems sketchy. We don’t know that m_data is the right size.

Maybe we can use AXTextMarkerGetLength? We could even make a helper function for span(AXTexrMarker).

Reply:
They check the length a few lines above so we know the size is right. However, your point stands that it'd be better to have a utility function to convert into a span.